### PR TITLE
[Contributions Dashboard] Wire up the instrumentation

### PR DESCRIPTION
**Pull request by cooltey**
**Phabricator:**
https://phabricator.wikimedia.org/T377049

### CODE REVIEW

File: app/src/main/java/org/wikipedia/analytics/eventplatform/ContributionsDashboardEvent.kt [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                    activeInterface,
+                    campaignId?.let { "campaign_id: ${CampaignCollection.getFormattedCampaignId(it)}, " }
+                        .orEmpty() + "donor_detected: ${Prefs.isDonor}",
+                    wikiId
+                )
+            }
+        }
+
+        fun anonSuffix(): String {
+            return if (AccountUtil.isLoggedIn) {
```
Comment (dbrant) [inline app/src/main/java/org/wikipedia/analytics/eventplatform/ContributionsDashboardEvent.kt:31]: Should this be the opposite? `!isLoggedIn`?

File: app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt [hunk 2]
```diff
@@ -136,7 +137,10 @@ class SuggestedEditsTasksFragment : Fragment() {
         binding.swipeRefreshLayout.setOnRefreshListener { refreshContents() }
 
         binding.errorView.retryClickListener = View.OnClickListener { refreshContents() }
-        binding.errorView.loginClickListener = View.OnClickListener { requestLogin.launch(LoginActivity.newIntent(requireContext(), LoginActivity.SOURCE_SUGGESTED_EDITS)) }
+        binding.errorView.loginClickListener = View.OnClickListener {
+            ContributionsDashboardEvent.logAction("login_click", "contrib_dashboard${ContributionsDashboardEvent.anonSuffix()}")
```
Comment (dbrant) [inline app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt:141]: Doesn't the schema already have an automatic field for `is_anon` and `is_temp`? (Why do we need extra logic to append `_anon` here, too?)

**Review by dbrant — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/wikipedia/analytics/eventplatform/ContributionsDashboardEvent.kt (hunk 1 above)
Comment (cooltey) [reply app/src/main/java/org/wikipedia/analytics/eventplatform/ContributionsDashboardEvent.kt]: Removed.

File: app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt (hunk 2 above)
Comment (cooltey) [reply app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt:141]: Done!

**Pull request merged**